netstandard/ref/System.Net.cs

```diff
@@ -663,14 +685,17 @@ public partial class HttpWebRequest : System.Net.WebRequest, System.Runtime.Seri
         public override System.IO.Stream EndGetRequestStream(System.IAsyncResult asyncResult) { throw null; }
         public System.IO.Stream EndGetRequestStream(System.IAsyncResult asyncResult, out System.Net.TransportContext context) { context = default(System.Net.TransportContext); throw null; }
         public override System.Net.WebResponse EndGetResponse(System.IAsyncResult asyncResult) { throw null; }
+        [System.ObsoleteAttribute("Serialization is obsoleted for this type.  http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
I don't think the Obsolete attributes should go to NS

Reply:
Obsolete messages are only interesting for reference assemblies as they are used by the compiler so if we want them they have to go here. It is a separate question of whether or not we feel these should be obsolete or not.

Reply:
I agree with @weshaggard. We should leave it in.

Reply:
Well, you are overriding non-obsolete member with an obsolete message. I don't think there is an easy way this to be ever visible to the end user (you'll get compiler warning for your code)